pkg/scheduler/testing/fake_lister.go

```diff
@@ -53,6 +53,23 @@ func (f FakePodLister) List(s labels.Selector) (selected []*v1.Pod, err error) {
 	return selected, nil
 }
 
+// Search returns []*v1.Pod matching all selectors
```

Comment:
looks just returns `selected []*v1.Pod` is ok

Reply:
Updated comment. Thanks.

Reply:
Sorry, I mean `Search `only needs to return `[]*v1.Pod`, `error` seems redundant.

Reply:
https://github.com/kubernetes/kubernetes/pull/73711/files#diff-6f5753a252698c11328724f8ed0b007cR51

error here is not optional. It is part of the interface definition.

Reply:
all right. thanks